### PR TITLE
Additional language GER => 'de'

### PR DESCRIPTION
**Pull request by DiWoWet**
I added up the German translations for the given commands as a means to use your code in my lessons :D Greetings from Germany and thanks for your awesome project! :)

### CODE REVIEW

File: scratch-vm/src/extensions/scratch3_tello/index.js [hunk 1]
```diff
@@ -25,216 +25,248 @@ const message = {
...
+        'fr': 'tourner de [X] degrés vers la gauche',
+        'de': 'drehe [X] Grad nach links'
     },
     flip: {
         'ja': '[DIRECTION]に宙返りする',
         'ja-Hira': '[DIRECTION]にちゅうがえりする',
         'en': 'flip in [DIRECTION]',
-        'fr': 'flip vers [DIRECTION]'
+        'fr': 'flip vers [DIRECTION]',
+        'de': 'Flip nach [DIRECTION]'
```
Comment (kebhr) [inline scratch-vm/src/extensions/scratch3_tello/index.js:108]: Perhaps the first letter is lowercase?
(If "Flip" is used as a noun and is capitalized, ignore my review.)
```suggestion
        'de': 'flip nach [DIRECTION]'
```

**Review by kebhr — COMMENTED** [review]
@DiWoWet
Vielen dank für die Übersetzung!

There is one place I would like you to check. I think everything else is fine and I can merge it as soon as I receive your reply.

File: scratch-vm/src/extensions/scratch3_tello/index.js (hunk 1 above)
Comment (DiWoWet) [reply scratch-vm/src/extensions/scratch3_tello/index.js:108]: I would really use it as a noun in this context. I also thought of "Drehung" but this rather means a circular movement on the x-axis and would thus be the wrong translation. Thanks for checking it after such a short time.

Comment (kebhr) [reply scratch-vm/src/extensions/scratch3_tello/index.js:108]: Understood. Now, I'm going to merge.

**Review by kebhr — APPROVED** [review]

**Comment by kebhr** [comment]
Within a few days I will release new binaries including German translation. Until then, please be patient.

**Comment by kebhr** [comment]
It is my true desire that this project will benefit education. A project is only effective when it is used. Thank you very much!

**Pull request merged**